cmake/AddFdbTest.cmake

```diff
@@ -91,6 +91,9 @@ function(add_fdb_test)
   # if the value was undefined before, it will still be
   # undefined.
   math(EXPR assigned_id "${test_idx} - 1")
+  math(EXPR test_file_idx "${CURRENT_TEST_FILE_INDEX} + ${NUM_TEST_FILES}")
```

Comment:
the only thing you should change here is the `math`-expression for `test_idx`

Reply:
Will do.  I kept the old test_idx just so when printing tests you get a contiguous count, but that doesn't happen when tests are skipped anyway